### DISCUSSION TITLE
What is the right way to manage workspaces and bunlock?

### DISCUSSION THREAD

**Discussion by 0xMySt1c** [oven-sh/bun#2636] [Q&A] [answered]
I have a parent project and nextjs workspace.
When I bun add from the nextjs folder, it will create a bunlock inside the workspace.
But when I bun install from top level: no bunlock is created in the workspace (only at the top level)
What is the right way to go about this? Should I be using --no-save on the workspace?
Or is this related to the link utility in bun?

**Comment by Jarred-Sumner** [comment, marked as answer]
I think this is https://github.com/oven-sh/bun/issues/2592

**Reply by 0xMySt1c** [reply]
thank you. loving the bun experience so far.🙏
